packages/core/src/types.ts

```diff
@@ -87,7 +87,14 @@ export interface DeanonymizeOptions extends RegistrationOptions {
   email?: string
   password?: string
 }
-export interface ProviderOptions extends RegistrationOptions, RedirectOption {}
+
+export interface CommonProviderOptions extends RegistrationOptions, RedirectOption {}
+export interface WorkOsOptions {
+  connection?: string
+  organization?: string
+  provider?: string
```

Comment:
Is at least one of these properties required to be set?

If yes, is there a way to make typescript understand that?

Reply:
no, if none of these is set, `hasura-auth` will fall back to a default value. If no default value in `hasura-auth`, it will fail.
But with these options, it is possible to let the frontend dev decide which connection/org/provider to use